app/src/main/java/org/p2p/wallet/auth/interactor/AuthInteractor.kt

```diff
@@ -27,6 +26,7 @@ private const val KEY_ENABLE_FINGERPRINT_ON_SIGN_IN = "KEY_ENABLE_FINGERPRINT_ON
  * If we decide to add pin code validation to the backend, we can remove one type hash
  * and make validation via pin code without decrypting hash
  * */
+// todo: tech-debt, split biometric and pin responsibility to separate classes
```

Comment:
biometric and pin code cannot be splitted because we are using both params to store/edit/read/create hash, salt and other stuff to work with secure storage

Reply:
then how about extracting shared prefs to a repository of some data layer class? 
i'll remove it then

Reply:
Let's talk on Android Sync about that.